PacketDotNet/GrePacket.cs

```diff
@@ -70,7 +79,14 @@ public GrePacket(ByteArraySegment byteArraySegment, Packet parentPacket)
         public EthernetType Protocol => (EthernetType) EndianBitConverter.Big.ToUInt16(Header.Bytes,
                                                                                        Header.Offset + GreFields.FlagsLength);
 
-        public int Version => Header.Bytes[2] & 0x7;
+        public int Version => Header.Bytes[Header.Offset + 1] & 0x7;
```

Comment:
Now that you're editing this, can you also add a setter?

Reply:
done